pro_1.py

```diff
@@ -0,0 +1,92 @@
+import json
+from re import A
+import re
```

Comment:
this library is used in the imports twice why?
what is A? and where is it used?
why don't you do `from re import findall`